### PR TITLE
APP-33383 allow project override for disaster recovery

### PR DESCRIPTION
**Pull request by mccall114**
This change is [Reviewable](https://reviewable.io/reviews/pendo-io/appwrap/105)

### CODE REVIEW

File: appinfoflex.go [hunk 1]
```diff
@@ -28,6 +28,14 @@ func InternalNewAppengineInfoFromContext(c context.Context) AppengineInfo {
 var NewAppengineInfoFromContext = InternalNewAppengineInfoFromContext
 
 func (ai AppengineInfoFlex) AppID() string {
+	if project := os.Getenv("GOOGLE_CLOUD_PROJECT_OVERRIDE"); project != "" {
```
Comment (a13m) [inline appinfoflex.go:31]: I might prefer a different name for this variable now that it's not overriding the project for all resources.  Maybe call it `GOOGLE_CLOUD_DATA_PROJECT` or something.

File: appinfoflex.go [hunk 2]
```diff
@@ -28,6 +28,14 @@ func InternalNewAppengineInfoFromContext(c context.Context) AppengineInfo {
...
+		return project
+	}
+
+	return os.Getenv("GOOGLE_CLOUD_PROJECT")
+}
+
+func (ai AppengineInfoFlex) AppIDHosted() string {
```
Comment (jshore1296) [inline appinfoflex.go:38]: Along the same lines as what andy said, I think cleaner function names would be good here.  From a dev perspective it is not clear which of these you should be using if you need to make a new call somewhere.  Good names are hard :( 

Something that conveys that one of these always returns the native project as in the project you're executing in.  The other one returns where your data is.

**Comment by a13m** [comment]
[merge]

**Comment by pendo-ops** [comment]
All JIRA issue transitioned/labeled .
* [APP-33383](https://pendo-io.atlassian.net/browse/APP-33383): Jira Issue APP-33383: Changed to Master


Branch APP-33383-appid-override-for-dr deleted

**Pull request merged**